0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/DeployCommand.scala

```diff
@@ -0,0 +1,99 @@
+package com.snowplowanalytics.iglu.ctl
+
+// java
+import java.io.File
+
+// scalaz
+import scalaz._
+
+// json4s
+import org.json4s._
+import org.json4s.jackson.JsonMethods.{asJsonNode, fromJsonNode}
+
+// iglu scala client
+import com.snowplowanalytics.iglu.client.{Resolver, SchemaKey}
+import com.snowplowanalytics.iglu.client.validation.ValidatableJsonMethods.validateAgainstSchema
+import com.snowplowanalytics.iglu.client.repositories.{HttpRepositoryRef, RepositoryRefConfig}
+
+// this project
+import FileUtils.getJsonFromFile
+
+
+case class DeployCommand(config: File) extends Command.CtlCommand {
+
+  implicit val formats: DefaultFormats = DefaultFormats
+
+  /**
+    * Primary method of static deploy command
+    * Performs usual schema workflow at once, per configuration file
+    * Short-circuits on first failed step
+    */
+  def process(): Unit = {
+    getJsonFromFile(config) match {
+      case Success(configDoc) =>
+        SchemaKey.parseNel("iglu:com.snowplowanalytics.iglu/igluctl_config/jsonschema/1-0-0") match {
+          case Success(schemaKey) =>
+            implicit val resolver: Resolver = getResolver()
+            resolver.lookupSchema(schemaKey) match {
+              case Success(configSchema) =>
+                validateAgainstSchema(asJsonNode(configDoc).get("data"), configSchema) match {
+                  case Success(doc) =>
+                    val allConfig: JValue = fromJsonNode(doc)
```

Comment:
This should be a separate function extracting some configuration class from JSON.